corefoundation_1.10.go

```diff
@@ -239,8 +239,8 @@ func Convert(ref C.CFTypeRef) (interface{}, error) {
 				return nil, err
 			}
 			results = append(results, v)
-			return results, nil
 		}
+		return results, nil
```

Comment:
this is behavioral change?

Reply:
although, it does seem like the old behavior was wrong. how did you find this? staticcheck doesnt report an error here

Reply:
it seems reasonable to me but maybe worth to do a test darwin build just to make sure..

Reply:
static check reported this for me, i'll do some test builds with the revendored libs

Reply:
cool, I made a ticket for vendoring everything in. But idk where it went during the squad migration lol